101-aci-storage-file-share/README.md

```diff
@@ -0,0 +1,35 @@
+# Create a Storage Account and a File Share
+
+Create a Storage Account and a File Share via az-cli in Container Instance - <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-aci-storage-file-share%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-aci-storage-file-share%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template demonstrates createing Storage File Share via az-cli and Azure Container Instance, modify the template to create blob or other similar storage operations. Azure Container Instance and az-cli can be used to create services not directly available in Azure Resource Manager Template.
+
+`Tags: Azure Container Instance, az-cli`
+
+## Solution overview and deployed resources
+
+This is an overview of the solution
+
+The following resources are deployed as part of the solution
+
++ **Storage Account**: Storage account for the file share
++ **Azure Container Instance**: Azure Container Instance, where the az-cli is exectued to create the file share
++ **File share**: [Azure File share](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction) accessiable via SMB protocol
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document or follow the instructions for command line deployment using the scripts in the root of this repo.
+
+## Usage
```

Comment:
remove this

Reply:
remove the "Usage" title or the entire "Usage" part?

Reply:
The title - since there's nothing in that section